packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div :class="$style['c-restaurantCarouselWrapper']">
+        <heading :class="$style['c-restaurantCarousel-heading']" />
+        <text-block :class="$style['c-restaurantCarousel-desc']" />
+        <div :class="$style['c-restaurantCarousel']">
+            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
+            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
+            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
+        </div>
+    </div>
+</template>
+
+<script>
+import TextBlock from './Text.vue';
+import Heading from './Heading.vue';
+import RestaurantCarouselCard from './RestaurantCarouselCard.vue';
+
+export default {
+    components: {
+        Heading,
+        TextBlock,
+        RestaurantCarouselCard
+    }
+};
+</script>
+
+<style lang="scss" module>
+
+$breakpoints: (medium: 520px, large: 960px);
+
+.c-restaurantCarouselWrapper {
+    margin: auto;
+    max-width: 1208px;
```

Comment:
Any reason why this is 1208px? It might be worth extracting it into a variable so that you can give it a descriptive name?

Reply:
Removed